Stripe/StripeiOSTests/STPAddressTests.swift

```diff
@@ -0,0 +1,532 @@
+//  Converted to Swift 5.8.1 by Swiftify v5.8.28463 - https://swiftify.com/
```

Comment:
should we delete these lines?

Reply:
I'm in favor of keeping them, considering these are just tests and it helps explain why the code is so weird